core/src/main/java/greencity/repository/UserRepo.java

```diff
@@ -79,4 +79,37 @@
      */
     @Query("SELECT profilePicturePath FROM User WHERE id=:id")
     Optional<String> getProfilePicturePathByUserId(Long id);
+
+    /**
+     * Get all user friends{@link User}.
+     *
+     * @return list of {@link User}.
+     */
+    @Query(value = " SELECT * FROM public.users_friends "
+        + " LEFT JOIN public.users ON public.users.id = public.users_friends.friends "
+        + " WHERE public.users_friends.user_id = :userId ", nativeQuery = true)
+    List<User> getAllUserFriends(Long userId);
+
+    /**
+     * Delete friend {@link User}.
+     */
+    @Modifying
+    @Query(value = "DELETE FROM public.users_friends WHERE user_id= :userId AND friends= :friendId", nativeQuery = true)
```

Comment:
I usually put `nativeQuery = true` at the beginning, so when I'm reading the code I understand from the start which notation is used.